crates/astria-conductor/src/config.rs

```diff
@@ -60,6 +60,19 @@ pub struct Config {
     /// The execution commit level used for controlling how blocks are sent to
     /// the execution layer.
     pub execution_commit_level: CommitLevel,
+
+    /// Websocket URL of Ethereum L1 node
+    #[cfg(feature = "optimism")]
```

Comment:
This is problematic in the config because we are testing to ensure that the local example env is in sync with the config. So it will either be in sync with it (if `--features optimism`) or not. :-/

Reply:
yeah, the unit test fails if `--features="optimism"` is not enabled :/ is there an easy way around this? should i create one unit test for the feature and one without (and two example envs?)